recipe/meta.yaml

```
@@ -10,16 +17,24 @@ source:
   sha256: 65e086708ae56670676b7b2340ad82b901d8c9993d1241a839c8990bdb8d6212
   patches:
       - patches/0000-Consolidated-build-fixes-for-XLA.patch  # [not win]
+      - patches/0001-Add-missing-typename.patch
 
 build:
-  number: 0
+  number: {{ build }}
   # s390x is missing bazel.
   skip: true  # [s390x or py<310]
+  skip: true  # [(cuda_compiler_version != "None") and aarch64]
+  skip: true  # [(cuda_compiler_version != "None") and win]
+  skip: true  # [cuda_compiler_version == "11.8"]
+  skip: true  # [skip_cuda_prefect and (gpu_variant or "").startswith('cuda')]
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}              # [cuda_compiler_version != "None"]
```

```suggestion
    - {{ compiler('cuda') }}              # [(gpu_variant or "").startswith('cuda')]
```
here and below. If we add cuda_compiler_version into the base conda_build_config.yaml at some point, it'll break this feedstock as currently written